examples/deepseek_vl_example.py

```diff
@@ -0,0 +1,63 @@
+import os
+import subprocess
+
+import torch
+from PIL import Image
+
+# The assets are located at `s3://air-example-data-2/vllm_opensource_llava/`.
+# You can use `.buildkite/download-images.sh` to download them
+from vllm import LLM, SamplingParams
+
+sample_params = SamplingParams(temperature=0, max_tokens=1024)
+model = "deepseek-ai/deepseek-vl-7b-chat"
+model = "deepseek-ai/deepseek-vl-1.3b-chat"
+prompt = "You are a helpful language and vision assistant." \
+    "You are able to understand the visual content that the user provides," \
+    "and assist the user with a variety of tasks using natural language.\n" \
+    "User: <image_placeholder> Describe the content of this image.\nAssistant:"
+
+
+def run_deepseek_vl():
+    llm = LLM(model=model,
+              max_model_len=3072,
+              enforce_eager=True,
+              dtype=torch.bfloat16)
+
+    image = Image.open("images/stop_sign.jpg")
```

Comment:
Please note that images can now be downloaded automatically when accessed via `vllm.assets.image.ImageAsset`.

Reply:
OK, the changes have been made